pmd-core/src/main/java/net/sourceforge/pmd/renderers/XMLRenderer.java

```diff
@@ -152,6 +152,17 @@ public void end() throws IOException {
                 writer.write(buf.toString());
             }
         }
+        
+        // config errors
+        for (final Report.ConfigurationError ce : configErrors) {
+            buf.setLength(0);
+            buf.append("<configerror ").append("rule=\"");
```

Comment:
I'm wondering whether we have documented the XML report format somewhere.... We are adding now a new element, which might break other tools. I think maven-pmd-plugin is safe, since it doesn't rely on that renderer, but maybe the Jenkins plugin does?